src/Main/MainGui.cpp

```diff
@@ -48,6 +48,7 @@
 #include <QApplication>
 #include <QLocale>
 #include <QMessageBox>
+#include <QtQuick/QQuickWindow>

```

Comment:
Do we really need this? If not it should be dropped. If yes then it should be visible for MSVC only. For MSYS/MinGW it's not needed either.

Reply:
No, I think that was leftover from when I was fighting with the OpenGL issues in Qt6. I'll remove it.